roles/2-common/tasks/packages.yml

```diff
@@ -54,6 +54,7 @@
       - ntfs-3g
       #- openssh-server    # ssh (Raspbian) or openssh-server (other OS's) already installed by 1-prep's roles/sshd/tasks/main.yml
       - pandoc
+      - pastebinit
```

Comment:
Just FYI pastebinit it was already added to IIAB here (lines 36 & 57 of this file) about a month ago:
https://github.com/iiab/iiab/blob/master/roles/2-common/tasks/packages.yml#L36-L57

(As part of PR #1700, as manual usage of pastebinit is so vital across all kinds of diff troubleshooting situations!)